handlers/propertyInspectionReports.js

```diff
@@ -27,5 +33,31 @@ function propertyInspectionReports(request, reply) {
 
 }
 
-module.exports = propertyInspectionReports;
+function postHandler(request, reply) {
+  var payload = request.payload;
+  var userId = 6; //Temporary hardcoded - userid currently can't be passed via params
```

Comment:
Please don't ever hardcode things like this. If its hardcoded, it means the feature certainly doesn't work, so there shouldn't be a PR for adding it to the software release.